consensus/src/config.rs

```diff
@@ -34,6 +37,34 @@ pub struct Config {
     pub max_round: Round,
 }
 
+#[derive(Clone)]
+pub struct LocalIO<D: Data, DP: DataProvider<D>, FH: FinalizationHandler<D>, UB: Write, UR: Read> {
```

Comment:
I think this should rather be in `member.rs`. The fields also wouldn't have to be `pub`, which is nice.

Reply:
Is the user supposed to provide their implementation of `UB` and `UR`? If so, then this fact should be noted in the `aleph-bft-types` package, possibly by moving this whole struct there?

Reply:
The types are fortunately from `std`, so I don't think that is necessary.

Reply:
on second thought (after a short chat with @timorl ), maybe not...

Reply:
Pff, sry, not args ofc, but the fields of `LocalIO`.

Reply:
done